style.css

```diff
@@ -0,0 +1,64 @@
+.nav-fixed-adjust {
+  margin-top: 4rem;
+}
+
+.landing-img {
+  width: 100%;
+  height: 50vh;
+  object-fit: cover;
+}
+
+.categories,
+.new-arrivals {
+  gap: 1rem;
+  padding: 2rem;
+}
+
+.categories {
+  justify-items: center;
+}
+
+.category-img {
+  height: 100%;
+  border-radius: 4px;
+}
+
+.new-arrivals .card {
+  justify-self: center;
+  min-width: 30vw;
+}
+
+.new-arrivals .card-img {
+  width: 100%;
+  border-radius: 4px;
+}
+
+.new-arrivals .c-badge-right {
+  z-index: 2;
+}
+
+.nav-fixed {
+  z-index: 9999;
```

Comment:
Suggestion: it's not necessary to use this high value for `z-index`, `z-index: 5 or 10` may also work i guess

Reply:
Was just trying to be too sure that it stays on the top 😅, will change it.